### DISCUSSION TITLE
i did not cheat to get this position

### DISCUSSION THREAD

**Discussion by aeolus-1** [aeolus-1/triangleGame.github.io#66] [General]
<img width="1063" alt="Screen Shot 2022-04-22 at 2 49 45 pm" src="https://user-images.githubusercontent.com/102002218/164609867-c31bfb52-67d9-4462-9edd-1a11d35c8094.png">
a ball squashed me

**Comment by Quantaviousdingleberry420** [comment]
that happens sometimes, i fell through the map because a ball crushed me
